Source/PLCrashLogWriter.m

```diff
@@ -54,6 +54,10 @@
 #import "PLCrashSysctl.h"
 #import "PLCrashProcessInfo.h"
 
+#if TARGET_OS_IOS
+#import <UIKit/UIKit.h>
```

Comment:
As far as I remember, it was specially removed some time ago. (please re-check the history)

Reply:
Yes, this was removed and the ProcessInfo was used to get the version number instead of UIDevice.  As far as I am aware there is no other way to retrieve the battery level than from UIDevice.  I am adding this back in only for iOS devices so that I can report back the battery level at the time of a crash.

https://github.com/microsoft/plcrashreporter/pull/37